app/controllers/parcels_controller.rb

```diff
@@ -28,7 +30,7 @@ def show
   private
 
   def parcel_params
-    params.require(:parcel).permit(:weight, :width, :depth, :height, :name)
+    params.require(:parcel).permit(:weight, :width, :depth, :height, :name, sender_info_attributes: [:email, :contact_name, :zip_code, :address, :city, :phone_number, :company_name, :other_info, :residential], recipient_info_attributes: [:email, :contact_name, :zip_code, :address, :city, :phone_number, :company_name, :other_info, :residential])
```

Comment:
Nie do końca wiem, jak to zrobić, żeby to miało sens.
```params.require(:parcel).permit(
  :weight, :width, :depth, :height, :name,
  sender_info_attributes: [:email, :contact_name, :zip_code, :address, :city,
                           :phone_number, :company_name, :other_info, :residential],
  recipient_info_attributes: [:email, :contact_name, :zip_code, :address, :city,
                              :phone_number, :company_name, :other_info, :residential])```
Tak wygląda trochę śmiesznie i nieregularnie. Inne warianty są jeszcze bardziej nieczytelne. Może powinienem stworzyć array "address_attributes" i jego podawać do tego hasha, zamiast je wypisywać tak bez sensu? I jeżeli mi się rozejdą między dwoma modelami, to wtedy się martwić, jak to przedstawić?